server/game/gamesteps/sundown/discardtohandsizeprompt.js

```diff
@@ -0,0 +1,53 @@
+const BaseStep = require('../basestep');
+
+class DiscardToHandSizePrompt extends BaseStep {
+    constructor(game) {
+        super(game);
+        this.players = this.game.getPlayersInFirstPlayerOrder();
+    }
+
+    continue() {
+        while(this.players.length) {
+            let currentPlayer = this.players.shift();
+            if(! currentPlayer.isOverHandsizeLimit()) {
+                this.game.addMessage('{0} already meets their hand size limit', currentPlayer);
```

Comment:
I am not sure if we should add message even if the player meets the hand size limit. The discard down is not very often so I think only the message when it happens is needed.

Reply:
:heavy_check_mark: